README.md

```diff
@@ -234,6 +234,15 @@ channel.unsubscribe('event', listener)
 channel.unsubscribe()
 ```
 
+#### Subscribe to connection state change
+```python
```

Comment:
I think we should add `from ably.realtime.connection import ConnectionState` just so that users know where to get the state enum from